backend/app/services/subscription/follow_service.py

```
@@ -0,0 +1,1244 @@
+            Success message
+
+        Raises:
+            HTTPException: If not owner or share not found
+        """
+        # Verify ownership
+        subscription = (
+            db.query(Kind)
+            .filter(
+                Kind.id == subscription_id,
+                Kind.kind == "Subscription",
+                Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)